### PR TITLE
C#: Use .NET 6

### PR DESCRIPTION
**Pull request by hvitved**

### CODE REVIEW

File: csharp/extractor/Semmle.Extraction.CSharp/Entities/UserOperator.cs [hunk 1]
```diff
@@ -178,6 +178,9 @@ public static bool OperatorSymbol(string methodName, out string operatorName)
         /// <returns>The converted name.</returns>
         private static string OperatorSymbol(Context cx, IMethodSymbol method)
         {
+            if (method.ExplicitInterfaceImplementations.Any())
+                return OperatorSymbol(cx, method.ExplicitInterfaceImplementations.First());
```
Comment (tamasvajk) [inline csharp/extractor/Semmle.Extraction.CSharp/Entities/UserOperator.cs:182]: Why is this change needed?

**Review by tamasvajk — APPROVED** [review]
looks plausible

File: csharp/extractor/Semmle.Extraction.CSharp/Entities/UserOperator.cs (hunk 1 above)
Comment (hvitved) [reply csharp/extractor/Semmle.Extraction.CSharp/Entities/UserOperator.cs:182]: It has been an issue all along, but with the upgrade we ran into it in the tests, which we didn't before.

**Review by michaelnebel — APPROVED** [review]
Looks plausible to me!
I suppose the failing test are caused by a needed update in the semmle-code repo, which will be made in another PR?

**Comment by hvitved** [comment]
> I suppose the failing test are caused by a needed update in the semmle-code repo, which will be made in another PR?

Yes.
